docs/Installation/Installation.md

```diff
@@ -20,7 +20,7 @@ environment differently, read on!
 #### Required:
 
 * [GCC](https://gcc.gnu.org/) 7.0 or later,
-[Clang](https://clang.llvm.org/) 5.0 or later, or AppleClang 6.0 or later
+[Clang](https://clang.llvm.org/) 7.0 or later, or AppleClang 7.0 or later
```

Comment:
I think you need at least AppleClang10.0.1 to be comparable to clang 7.0.  Have you built with AppleClang? I will try to do so in a bit...

Reply:
I'm building with AppleClang 12 these days, and built with AppleClang 11 before. Haven't tried an older version for a long time.